### PR TITLE
Update Docker readme to match online copy

### PR DESCRIPTION
**Pull request by acodrington**
The copy in this file, and the copy viewable at https://hub.docker.com/r/octopusdeploy/tentacle, are synced manually. This commit just updates the repo copy to match the updated online copy.

Fixes internal issue [sc-44418]

### CODE REVIEW

**Review by geofflamrock — APPROVED** [review]
LGTM, matches the docker hub readme

File: docker/run-tests.ps1 [hunk 1]
```diff
@@ -20,12 +20,12 @@ Add-Type -Path (Join-Path (Get-Item ((Get-Package Octopus.Client).source)).Direc
 . .\common.ps1
 
 $networkName = "${ProjectName}_default"
```
Comment (cailyoung) [inline docker/run-tests.ps1:22]:
```suggestion
$networkName = "${ProjectName}-default"
```

May also be required?

File: docker/common.ps1 [hunk 2]
```diff
@@ -1,6 +1,9 @@
 function Get-IPAddress() {
     param ([ValidateNotNullOrEmpty()][string]$network, [ValidateNotNullOrEmpty()][string]$container)
-
+	
+	#debug
+	docker inspect $container
+
```
Comment (cailyoung) [inline docker/common.ps1:6]:
```suggestion
```

File: docker/common.ps1 [hunk 3]
```diff
@@ -1,6 +1,9 @@
 function Get-IPAddress() {
     param ([ValidateNotNullOrEmpty()][string]$network, [ValidateNotNullOrEmpty()][string]$container)
-
+	
+	#debug
+	docker inspect $container
+	
     $docker = (docker inspect $container | convertfrom-json)[0]
     return $docker.NetworkSettings.Networks.$network.IpAddress
```
Comment (cailyoung) [inline docker/common.ps1:8]:
```suggestion
    return $docker.NetworkSettings.Networks.$network.IPAddress
```

Comment (cailyoung) [reply docker/common.ps1:8]: 🤦

**Pull request merged**